server/models/PayoutMethod.ts

```diff
@@ -23,11 +23,13 @@ export enum PayoutMethodTypes {
 /** An interface for the values stored in `data` field for PayPal payout methods */
 export interface PaypalPayoutMethodData {
   email: string;
+  accountHolderName: string;
```

Comment:
Why adding this to `PaypalPayoutMethodData` and `OtherPayoutMethodData`? Don't we want that just for `BankAccountPayoutMethodData`?

Reply:
@Betree : The reason this was added is due to typescript errors such as; 

```
server/graphql/common/expenses.ts:632:26 - error TS2339: Property 'accountHolderName' does not exist on type 'PayoutMethodDataType'.
  Property 'accountHolderName' does not exist on type 'PaypalPayoutMethodData'.
```

I couldn't find a more elegant way to do this. 🤔

Reply:
The property doesn't exists on `PaypalPayoutMethodData` and `OtherPayoutMethodData`, so there's no reason to add it there. What we want is to make sure typescript understands that we're playing with a `BankAccountPayoutMethodData` in there. For example, by casting the value:

```ts
if (payoutMethod?.type === PayoutMethodTypes.BANK_ACCOUNT) {
  const payoutMethodData = <BankAccountPayoutMethodData>payoutMethod.data;
  if (payoutMethodData?.accountHolderName !== expenseData.fromCollective.legalName) {
    ...
  }
}
```

Reply:
Ah, that makes perfect sense. I need to brush up on my Typescript skills. 🤔